hint-report/file--.html

```diff
@@ -0,0 +1,2325 @@
+<!DOCTYPE html>
```

Comment:
[OPTIONAL] The same as with the vscode config files applies to this report from your linters.